frontends/election-manager/src/utils/election.test.ts

```diff
@@ -29,7 +29,7 @@ describe('generateOvervoteBallot', () => {
     });
     expect(overvoteBallot).toBeDefined();
 
-    const votes = overvoteBallot!.votes as VotesDict;
+    const { votes } = overvoteBallot!;
```

Comment:
This casting is no longer necessary because of the `Ballot` type I define in `frontends/election-manager/src/utils/election.ts` below